src/pages/docs/chat/integrations.mdx

```diff
@@ -0,0 +1,65 @@
+
+If you're looking to add moderation, Ably Chat already has [built-in moderation](/docs/chat/moderation) support for Hive, Tisane, and Bodyguard, and you can also [build custom moderation](/docs/chat/moderation/custom).
+
+This guide is also applicable if you are using Ably Chat with a language or platform that doesn't yet have an Ably Chat SDK, but does have an Ably Pub/Sub SDK.
+
+## Convert a Pub/Sub message to a Chat message <a id="pub-sub-to-chat"/>
+
+To convert a Pub/Sub message to a Chat message you can use the table below:
+
+| Ably Message Field | Chat Message Field | Transformation/Notes |
+|-------------------|-------------------|---------------------|
+| `clientId` | `clientId` | Direct mapping. |
+| `timestamp` | `timestamp` | In Pub/Sub, this is a UNIX timestamp in milliseconds since the epoch. In Chat, this is a Date object. |
+| `serial` | `serial` | Direct mapping. |
+| `data.text` | `text` | Data must be JSON-decoded first. |
+| `data.metadata` | `metadata` | Data must be JSON-decoded first. |
+| `extras.headers` | `headers` | Flat object, key-value pairs. |
+| `action` | `action` | `message.create`, `message.update`, `message.delete`, or `message.summary` |
+| `version.serial` | `version.serial` | Use message `serial` if no `version` is set. |
+| `version.timestamp` | `version.timestamp` | Use message `timestamp` if no `version` is set. |
+| key `"reaction:unique.v1"` in `annotations.summary` | `reactions.unique` | Message reactions of type unique |
+| key `"reaction:distinct.v1"` in `annotations.summary` | `reactions.distinct` | Message reactions of type distinct |
+| key `"reaction:multiple.v1"` in `annotations.summary` | `reactions.multiple` | Message reactions of type multiple |
+
+If you are using a language that has an Ably Pub/Sub SDK, you can use the `fromEncoded()` or `fromEncodedArray()` methods to decode the encoded message (the JSON or msgpack received) into a Pub/Sub message. Then you can apply the mapping from the table above to get the Chat message form the Pub/Sub message.
+
+If you are not using a language supported by an Ably Pub/Sub SDK, you will need to manually decode the message.
+
+Be aware that message actions are numeric values over the realtime channels, Pub/Sub REST API, and in integrations. Ably SDKs convert these numeric values to the corresponding string representations. If you do not use an Ably SDK, you will need to convert the numerical action values to their string representations.
+
+If possible, use an Ably Pub/Sub SDK with `fromEncoded()` or `fromEncodedArray()` to decode the message. This will ensure that the message is decoded correctly and all the fields are present, as well as convert things like numerical action values to their string representations. Details on how to do this are available in the [platform integrations documentation](/docs/platform/integrations/webhooks).
```

Comment:
This feels very close to the paragraph directly under the table - perhaps axe one?